src/frontends/tensorflow_common/src/op/binary_op.cpp

```diff
@@ -60,6 +73,56 @@ OutputVector translate_binary_op(const NodeContext& node) {
     });
 }
 
+OutputVector translate_mul_op(const NodeContext& node) {
+    default_op_checks(node, 2, {}, true);
+    auto lhs = node.get_input(0);
+    auto rhs = node.get_input(1);
+    auto result = make_shared<v1::Multiply>(lhs, rhs);
+    set_node_name(node.get_name(), result);
```

Comment:
I suspect this is better to do after `if` resolving complex types

Reply:
Done.